install/kubernetes/cilium/values.yaml

```
@@ -229,6 +229,9 @@ bgp:
     podCIDR: false
 
 bpf:
+  # -- Configure bpf root filesystem
```

Nit: “root filesystem” doesn't sound accurate, we don't have a tree of BPF filesystems :). How about “... the root of the BPF filesystem”, or even:
```suggestion
  # -- Configure the mount point for the BPF filesystem
```

---

thanks, the mount point for BPF filesystem sounds clearer definitely :100: